src/frontend/src/test-e2e/util.ts

```diff
@@ -88,6 +89,59 @@ export async function runInBrowserCommon(
   }
 }
 
+export async function startWebdriver(): Promise<ChildProcess | undefined> {
+  let webdriverProcess: ChildProcess | undefined;
+  let retryCount = 0;
+  let error;
+  while (webdriverProcess === undefined && retryCount < 10) {
+    try {
+      error = undefined;
+      webdriverProcess = await SeleniumStandalone.start();
+    } catch (e) {
+      // port may still be used from previous stopped webdriver, try again
+      error = e;
+      retryCount++;
+      await new Promise((resolve) => setTimeout(resolve, 1000));
+    }
+  }
+  if (error !== undefined) {
+    console.warn(
+      'selenium could not be started. Make sure you installed the required webdrivers ("npm run install-webdrivers")'
+    );
+    console.error(error);
+  }
+  return webdriverProcess;
+}
+
+export interface ScreenConfiguration {
+  screenType: "desktop" | "mobile";
+  windowSize: string;
+}
+
+export interface RunConfiguration {
+  screenConfiguration: ScreenConfiguration;
+}
+
+function parseScreen(): ScreenConfiguration {
+  switch (process.env.SCREEN) {
+    case "mobile":
+      return MOBILE_SCREEN;
+    case "desktop":
+      return DESKTOP_SCREEN;
```

Comment:
Can you explain what's happening with `ScreenConfiguration` and those environment vars? (not on GitHub, but in the code/README).

Reply:
I added some comments, especially on top where the screens are defined as constants. I hope that explains it well enough :-)

Reply:
Also as an entry point from the readme I added the package.json scripts, maybe that'll help as well